host-base.json

```diff
@@ -114,6 +114,7 @@
         "origin-node",
         "origin-hyperkube",
         "origin-clients",
+        "containernetworking-cni",
```

Comment:
Since this is an RPM dependency today, do you see value in explicitly listing it?

Reply:
Hrm, I guess not. My half awake brain added it because ``containernetworking-cni`` provides ``containernetworking-plugins``, but it should solve without explicit addition like this. I'll drop it from this PR.

Reply:
Done